### PR TITLE
feat: create header

### PR DESCRIPTION
**Pull request by Egorkostuck**
feat: create header

### CODE REVIEW

File: src/components/shared/userInfoHeader/config.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const colorGold = '#ffd700';
```
Comment (AdammmVV) [inline src/components/shared/userInfoHeader/config.ts]: use [theming colors](https://mantine.dev/theming/colors/)

File: src/constants/constants.ts [hunk 2]
```diff
@@ -2,3 +2,4 @@ export const errorMessageNicknameField = 'Данный ник уже занят,
 export const errorMessageEmptyField = 'Заполните поле';
```
Comment (AdammmVV) [inline src/constants/constants.ts]: use translate

File: src/components/widget/header/Header.tsx [hunk 3]
```diff
@@ -13,18 +24,27 @@ interface IProps {
 
 export const Header: FC<IProps> = ({ opened, toggle }) => {
   const isAuth = useAppSelector(getIsAuth);
+  const { width } = useViewportSize();
```
Comment (AdammmVV) [inline src/components/widget/header/Header.tsx]: separate the hooks with an empty line. If they are not connected in meaning

File: src/components/shared/userInfoHeader/UserInfoHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { useAppSelector } from 'hooks';
+import { getNickname, getPhotoURl, getBalance } from 'store';
+
+export const UserInfoHeader: FC = () => {
+  const photoUrl = useSelector(getPhotoURl);
+  const balance = useSelector(getBalance);
+  const nickname = useAppSelector(getNickname);
```
Comment (AdammmVV) [inline src/components/shared/userInfoHeader/UserInfoHeader.tsx]: use `useAppSelector`

File: src/components/shared/navigation/Navigation.tsx [hunk 5]
```diff
@@ -16,7 +16,7 @@ export const Navigation: FC = () => {
   const { t } = useTranslation();
```
Comment (AdammmVV) [inline src/components/shared/navigation/Navigation.tsx:17]: add emty line for hook

File: src/components/layout/defaultLayout/DefaultLayout.tsx [hunk 6]
```diff
@@ -18,6 +21,7 @@ import { getIsAuth } from 'store';
 export const DefaultLayout: FC = () => {
   const isAuth = useAppSelector(getIsAuth);
   const [opened, { toggle }] = useDisclosure();
+  const { width } = useViewportSize();
```
Comment (AdammmVV) [inline src/components/layout/defaultLayout/DefaultLayout.tsx]: add emty line for hook

File: src/components/widget/header/Header.tsx (hunk 3 above)
Comment (AdammmVV) [inline src/components/widget/header/Header.tsx]: try use [use-media-query](https://mantine.dev/hooks/use-media-query/#usage)

File: src/components/shared/userInfoHeader/config.ts (hunk 1 above)
Comment (Egorkostuck) [reply src/components/shared/userInfoHeader/config.ts]: Yes, I don't use the variable in question, I color scheme for the theme. I forgot to delete the file and the variable

**Pull request merged**